resources/ark/README.md

```diff
@@ -8,3 +8,61 @@ Ark is a tool to back up and restore Kubernetes resources and persistent volumes
 
 By default, Ark comes with GCP as a backup storage provider and no bucket set. With that configuration, the Ark server deployment scales down to 0 replicas because Ark cannot start without the proper configuration for the backup storage bucket. You can change this by providing proper credentials in the heptio-ark/ark secret and changing the configuration in config/default.
 
+## Usage
+
+> **NOTE:** The steps below assume you have [downloaded](https://github.com/heptio/ark/releases) the ark binary and added it to the environment variable `$PATH` .
+
+> **NOTE:** The steps below assume you have created a lambda function in the production namespace .
+
```

Comment:
Maybe we can combine these two notes in a `Prerequisites` section? The we would have something like this: 
````
### Prerequisites

- [Download](https://github.com/heptio/ark/releases) the ark binary and add it to the **$PATH** environment variable. 
- Create a lambda function in the `production` Namespace.

````